enclave/build.rs

```diff
@@ -0,0 +1,11 @@
+use std::path::Path;
+
+use libbpf_cargo::SkeletonBuilder;
+
+const SRC: &str = "./src/bpf/enclave.bpf.c";
```

Comment:
Will these always be unix style paths? I assume so due to the feature set.

Reply:
Yes, always. And I don't expect this project to run on any non-Linux system (Linux kernel is required).

Would you prefer to create a path with `join` anyway? I could do that, especially since I do that in `main.rs` and I'm just being inconsistent. ;)

Reply:
This is fine! Just wanted to make sure. Related: https://github.com/rancher-sandbox/enclave/pull/2#discussion_r644851214